app/models/adl_document_wrapper.rb

```diff
@@ -0,0 +1,49 @@
+class AdlDocumentWrapper < ::OAI::Provider::Model
```

Comment:
classdoc would be nice, even just a reference to other documentation if it's already covered elsewhere.